<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>advancedVDialog</class>
 <widget class="QDialog" name="advancedVDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#advancedVDialog{border-image: url(:/Picture3.png);}</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>119</y>
     <width>562</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="1" column="3">
     <widget class="QRadioButton" name="electronicButton">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 150);</string>
      </property>
      <property name="text">
       <string>Electronic</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QRadioButton" name="folkButton">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 150);</string>
      </property>
      <property name="text">
       <string>Folk</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QRadioButton" name="jazzButton">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 150);</string>
      </property>
      <property name="text">
       <string>Jazz</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QRadioButton" name="popButton">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 150);</string>
      </property>
      <property name="text">
       <string>Pop</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QRadioButton" name="rockButton">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 150);</string>
      </property>
      <property name="text">
       <string>Rock</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QRadioButton" name="classicalButton">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 150);</string>
      </property>
      <property name="text">
       <string>Classical</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="2" alignment="Qt::AlignLeft">
     <widget class="QLabel" name="label_3">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(51, 91, 143, 230);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Genre</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QRadioButton" name="metalButton">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 150);</string>
      </property>
      <property name="text">
       <string>Metal</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>561</width>
     <height>53</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <item row="0" column="2" rowspan="2">
     <widget class="QSpinBox" name="saturationspinBox">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QSlider" name="saturationSlider">
      <property name="minimumSize">
       <size>
        <width>260</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>260</width>
        <height>20</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="2" alignment="Qt::AlignLeft">
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(51, 91, 143, 230);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Saturation</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3" rowspan="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>562</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="4">
     <widget class="QRadioButton" name="adlonelyButton">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 150);</string>
      </property>
      <property name="text">
       <string>Lonely</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QRadioButton" name="adexcitedButton">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 150);</string>
      </property>
      <property name="text">
       <string>Excited</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QRadioButton" name="adhappyButton">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 150);</string>
      </property>
      <property name="text">
       <string>Happy</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="2" alignment="Qt::AlignLeft">
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(51, 91, 143, 230);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Emotion</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QRadioButton" name="adnervousButton">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 150);</string>
      </property>
      <property name="text">
       <string>Nervous</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QRadioButton" name="adrelaxedButton">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 150);</string>
      </property>
      <property name="text">
       <string>Relaxed</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QRadioButton" name="adblueButton">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 150);</string>
      </property>
      <property name="text">
       <string>Blue</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QRadioButton" name="adsadButton">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 150);</string>
      </property>
      <property name="text">
       <string>Sad</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>350</y>
     <width>248</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="adokButton">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="adcancelButton">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>201</y>
     <width>334</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(51, 91, 143, 230);
color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string>Style</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QRadioButton" name="complexButton">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 150);</string>
      </property>
      <property name="text">
       <string>Complex</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QRadioButton" name="quietButton">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 150);</string>
      </property>
      <property name="text">
       <string>Quiet</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>saturationSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>saturationspinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saturationspinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>saturationSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>404</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
